### ISSUE TITLE
Missing keyboard nav feautures

### ISSUE THREAD

**Issue by mskocik** [mskocik/svelty-picker#7]
Missing features list:

- [ ] `shiftKey` implementation
- [ ] timer navigation

**Comment by mskocik** [comment]
🥳
